### ISSUE TITLE
Can't create additional cards

### ISSUE THREAD

**Issue by bobbyreed** [bobbyreed/LoCVideoGameChallenge#1]
Unity throws these message:

This one all the time that the card editor is open:
* GUI Error: Invalid GUILayout state in CardEditorWindow view. Verify that all layout Begin/End calls match
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

These two when a new card is trying to be created; the creation fails
* Couldn't create asset file!
UnityEngine.StackTraceUtility:ExtractStackTrace ()
NueGames.NueDeck.Editor.CardEditorWindow:CreateNewCard () (at Assets/Plug-Ins/NueGames/NueDeck/Editor/CardEditorWindow.cs:170)
NueGames.NueDeck.Editor.CardEditorWindow:DrawAllCardButtons () (at Assets/Plug-Ins/NueGames/NueDeck/Editor/CardEditorWindow.cs:149)
NueGames.NueDeck.Editor.CardEditorWindow:OnGUI () (at Assets/Plug-Ins/NueGames/NueDeck/Editor/CardEditorWindow.cs:109)
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)
* UnityException: Creating asset at path  failed.
NueGames.NueDeck.Editor.CardEditorWindow.CreateNewCard () (at Assets/Plug-Ins/NueGames/NueDeck/Editor/CardEditorWindow.cs:170)
NueGames.NueDeck.Editor.CardEditorWindow.DrawAllCardButtons () (at Assets/Plug-Ins/NueGames/NueDeck/Editor/CardEditorWindow.cs:149)
NueGames.NueDeck.Editor.CardEditorWindow.OnGUI () (at Assets/Plug-Ins/NueGames/NueDeck/Editor/CardEditorWindow.cs:109)
UnityEditor.HostView.InvokeOnGUI (UnityEngine.Rect onGUIPosition) (at <7fef092428a04dc89d2aa6ad3ce5d856>:0)
UnityEditor.DockArea.DrawView (UnityEngine.Rect dockAreaRect) (at <7fef092428a04dc89d2aa6ad3ce5d856>:0)
UnityEditor.DockArea.OldOnGUI () (at <7fef092428a04dc89d2aa6ad3ce5d856>:0)
UnityEngine.UIElements.IMGUIContainer.DoOnGUI (UnityEngine.Event evt, UnityEngine.Matrix4x4 parentTransform, UnityEngine.Rect clippingRect, System.Boolean isComputingLayout, UnityEngine.Rect layoutSize, System.Action onGUIHandler, System.Boolean canAffectFocus) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, UnityEngine.Matrix4x4 worldTransform, UnityEngine.Rect clippingRect, System.Action onGUIHandler, System.Boolean canAffectFocus) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, System.Action onGUIHandler, System.Boolean canAffectFocus) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, System.Boolean canAffectFocus) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.IMGUIContainer.SendEventToIMGUIRaw (UnityEngine.UIElements.EventBase evt, System.Boolean canAffectFocus, System.Boolean verifyBounds) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.IMGUIContainer.SendEventToIMGUI (UnityEngine.UIElements.EventBase evt, System.Boolean canAffectFocus, System.Boolean verifyBounds) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.IMGUIContainer.HandleEvent (UnityEngine.UIElements.EventBase evt) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.CallbackEventHandler.HandleEventAtTargetPhase (UnityEngine.UIElements.EventBase evt) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.MouseCaptureDispatchingStrategy.DispatchEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcher.ApplyDispatchingStrategies (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, System.Boolean imguiEventIsInitiallyUsed) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEventQueue () (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcher.OpenGate () (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcherGate.Dispose () (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.EventDispatcher.Dispatch (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, UnityEngine.UIElements.DispatchMode dispatchMode) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.BaseVisualElementPanel.SendEvent (UnityEngine.UIElements.EventBase e, UnityEngine.UIElements.DispatchMode dispatchMode) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.UIElementsUtility.DoDispatch (UnityEngine.UIElements.BaseVisualElementPanel panel) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.UIElementsUtility.UnityEngine.UIElements.IUIElementsUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr, System.Boolean& eventHandled) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.UIEventRegistration.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.UIElements.UIEventRegistration+<>c.<.cctor>b__1_2 (System.Int32 i, System.IntPtr ptr) (at <875b37995cee43cdaeb0e3c5cc262e4c>:0)
UnityEngine.GUIUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr, System.Boolean& result) (at <2880c19c3b344d8cb20d5ad03e3efbe5>:0)

Needs tracing.

**Comment by bobbyreed** [comment]
This happened right after I added the rarity layers. Maybe try to remove those or comment them out and see if it will work.

**Comment by bobbyreed** [comment]
I did not figure out how to get rid of the errors or really the root. One set of the errors come from Unity and I can't trace the cause. The other is -- I think -- coming from duplicating the script structures in my 7/15 session. What I did figure out, however, is a workaround (queue fanfare). From here `Assets/Plug-Ins/NueGames/NueDeck/Data/Cards/' I can duplicate the prefabs to make shells. I edit each prefeb directly or pull them up -- once cloned -- in the card editor.
